File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int n = Integer.parseInt(st.nextToken()); // 저장된 사이트 주소의 수
        int m = Integer.parseInt(st.nextToken()); // 비밀번호를 찾으려는 사이트 주소의 수
        Map<String, String> site = new HashMap<>();

        for(int i=0; i<n; i++) {
            st = new StringTokenizer(br.readLine());
            String url = st.nextToken();
            String password = st.nextToken();
            site.put(url, password);
        }

        for(int i=0; i<m; i++) {
            String url = br.readLine();
            System.out.println(site.get(url));
        }


    }
}

File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.kt
fun main() = with(System.`in`.bufferedReader()) {
    val (n, m) = readLine().split(" ").map { it.toInt() }
    val sb = StringBuilder()
    val map = HashMap<String, String>()
    repeat(n) {
        val (site, password) = readLine().split(" ")
        map.put(site, password)
    }
    repeat(m) {
        sb.appendLine(map[readLine()])
    }
    print(sb)
}